security/rbac/README.md

```diff
@@ -133,6 +133,30 @@ confluent iam rolebinding create --principal User:$CLIENT --role ResourceOwner -
 # REST Proxy Admin: no additional administrative rolebindings required because REST Proxy just does impersonation
 
 # Producer/Consumer
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+```
+
+
+## KSQL
+
+* [delta_configs/ksql-server.properties.delta](delta_configs/ksql-server.properties.delta)
+* Role bindings:
+
+```bash
+# KSQL Server Admin
```

Comment:
Fwiw, the recently blessed term (as of today) is "service principal".

Reply:
@JimGalasyn , @spena -- as we discussed in Slack, this line should actually be `user principal`, not `service principal`.  I've filed a separate jira DEVX-960 to use proper terminology for all services, not just KSQL. Thanks!